<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmLED</class>
 <widget class="QWidget" name="FrmLED">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QExtLed" name="led" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QExtLed" name="led_2" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="QExtLed" name="led_3" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="QExtLed" name="led_4" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QExtLed" name="led_5" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QExtLed" name="led_6" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="QExtLed" name="led_7" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="QExtLed" name="led_8" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QExtLed" name="led_9" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="QExtLed" name="led_10" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="QExtLed" name="led_11" native="true"/>
      </item>
      <item row="2" column="3">
       <widget class="QExtLed" name="led_12" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QExtLed</class>
   <extends>QWidget</extends>
   <header>qextLed.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
